monochart/templates/_common_metadata_labels.tpl

```diff
@@ -51,6 +51,6 @@ Labels to use in:
   statefulset.spec.volumeClaimTemplates.labels
 */}}
 {{- define "common.labels.short" -}}
-app.kubernetes.io/name: {{ include "common.name" . }}
```

Comment:
Same comment as above, I believe `common.name` for `app.kubernetes.io/instance`.